BeatSaberHTTPStatus/Plugin.cs

```diff
@@ -243,11 +243,12 @@ public class Plugin : IPlugin {
 
 				gameStatus.staticLights = playerSettings.staticLights;
 				gameStatus.leftHanded = playerSettings.leftHanded;
-				gameStatus.swapColors = playerSettings.swapColors;
 				gameStatus.playerHeight = playerSettings.playerHeight;
 				gameStatus.disableSFX = playerSettings.disableSFX;
+                gameStatus.reduceDebris = playerSettings.reduceDebris;
```

Comment:
Use tabs for indentation.

Reply:
Also,  good catch.

Reply:
I'm not sure how to go about doing this as visual studio isn't showing this to be an issue and Github Desktop isn't detecting that any changes have been made even when redoing the line indentation using tabs.

Reply:
Don't worry about my last comment, found the issue with my visual studio settings and have pushed a new commit with the changes requested.